examples/cub/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# Copyright (c) 2017, David Hirvonen
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cub)
+
+# Test CMake cub installation, even if find_package(CUDA) fails
+hunter_add_package(cub)
+find_package(cub CONFIG REQUIRED)
+
+find_package(CUDA QUIET REQUIRED)
```

Comment:
Got error:
```
CMake Error at /.../cmake-3.10/Modules/FindCUDA.cmake:682 (message):
  Specify CUDA_TOOLKIT_ROOT_DIR
Call Stack (most recent call first):
  CMakeLists.txt:17 (find_package)
```

I think for the start we can do just:
```cmake
hunter_add_package(cub)
find_package(cub CONFIG REQUIRED)

if(NOT TARGET cub::cub)
  message(FATAL_ERROR "Target not found")
endif()
```

Reply:
Done.